src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +145,11 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Gets this player's {@link TabList}.
+     * 
+     * @return This player's TabList.
+     */
+    TabList getTabList();
```

Comment:
All properties in the [TabList](https://github.com/SpongePowered/SpongeAPI/pull/433/files#diff-6917926cab088718af5ac88af7ef22b8R38) interface can be set so this makes no sense.